recipes/CatGifsReactHooks/src/Main.purs

```diff
@@ -55,33 +39,34 @@ main = do
 mkCatGifs :: Component {}
 mkCatGifs = do
   component "CatGifs" \_ -> React.do
-    (gifState /\ dispatch) <- useAffReducer Loading reducer
```

Comment:
Removed the use of `useAffReducer` here. The version of `react-basic-hooks` that's in the package set we're using is `v6.0.0`, and it seems that version doesn't export the `mkAffReducer` that's needed to construct an `AffReducer` 🙈 . This lead me to discover `useResetToken`, which the author seems to recommend over `useAffReducer` anyways, and it ends up being less code and a bit simpler 👍 .